compile/compile.go

```diff
@@ -223,6 +224,12 @@ func (c *Compiler) WithMetadata(metadata *map[string]interface{}) *Compiler {
 	return c
 }
 
+// WithRoots sets bundle roots in the .manifest
```

Comment:
Nit: Can we update the description to something like `WithRoots sets the roots to include in the output bundle manifest.`

Reply:
Thanks for the comment. Updated.